action.yml

```diff
@@ -19,6 +19,10 @@ inputs:
   awsAccount:
     description: 'AWS Account ID'
     required: true
+  useLatestTemplate:
+    description: 'Use the latest template configuration'
+    required: false
+    default: 'true'
```

Comment:
The default is `false` because we want always to get from the tag's commit SHA?

Reply:
actually the default is true

Reply:
yeah, it can be, i dont mind